lib/rubocop/cop/rspec/expect_change.rb

```diff
@@ -39,7 +39,7 @@ class ExpectChange < Base
 
         # @!method expect_change_with_arguments(node)
         def_node_matcher :expect_change_with_arguments, <<-PATTERN
-          (send nil? :change ({const send} nil? $_) (sym $_))
+          (send nil? :change $({const send} ...) (sym $_))
```

Comment:
Do we need to limit the argument of change at all?
Could be just `(send nil? :change $_ (sym $_))`

Reply:
This way it would also match `change(@ivar, :method)`, `change($global, :method)`, `change(::Const, :count)` and other possible arguments.
I don't have a definitive list in my head, can there be any false positives? Something like `change(-> {}, :call)`?
👍 for such a change granted a few more examples are added.
@tejasbubane Are you up for that, or would you prefer us to file a ticket for the future?

Reply:
Wondering if `${const send}` can be a drop-in replacement for`$({const send} ...)` 🤔

Reply:
Changed as per @Darhazer suggestion.

Reply:
@bquorning What do you think?